### PR TITLE
feat(bigtable): Add integer support to RowFilter#value and ValueRange

### PR DESCRIPTION
**Pull request by quartzmo**
* Encode Integer arguments as 64-bit signed big-endian integer data.

closes: #8179

### CODE REVIEW

File: google-cloud-bigtable/lib/google/cloud/bigtable/convert.rb [hunk 1]
```diff
@@ -77,6 +77,18 @@ def time_to_timestamp time
...
         end
+
+        ##
+        # Converts an Integer to 64-bit signed big-endian integer data.
+        # Returns a string argument unchanged.
+        #
+        # @param value [String, Integer]
+        # @return [String]
+        #
+        def integer_to_signed_be_64 value
```
Comment (dazuma) [inline google-cloud-bigtable/lib/google/cloud/bigtable/convert.rb:88]: Should have unit tests for this in test/google/cloud/bigtable/convert_test.rb.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-bigtable/lib/google/cloud/bigtable/convert.rb (hunk 1 above)
Comment (quartzmo) [reply google-cloud-bigtable/lib/google/cloud/bigtable/convert.rb:88]: Added.

**Review by dazuma — APPROVED** [review]

**Comment by quartzmo** [comment]
@billyjacobson ping!

File: google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb [hunk 2]
```diff
@@ -65,7 +67,9 @@ def initialize
         ##
         # Sets the row range with the lower bound.
         #
-        # @param value [String] The value. Required.
+        # @param value [String, Integer] The value. Required. If the argument
+        #   is an Integer, it will be encoded as a 64-bit signed big-endian
```
Comment (billyjacobson) [inline google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb:71]: Could these changes break existing users by auto converting the value?

**Review by billyjacobson — COMMENTED** [review]
Looks good from a Bigtable perspective, left one clarifying question

File: google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb (hunk 2 above)
Comment (quartzmo) [reply google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb:71]: The `Integer` argument type was not previously declared in the docs or supported.

Comment (quartzmo) [reply google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb:71]: @billyjacobson Does this sound OK to you?

Comment (billyjacobson) [reply google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb:71]: Alright that sounds good!

Comment (quartzmo) [reply google-cloud-bigtable/lib/google/cloud/bigtable/value_range.rb:71]: @billyjacobson Awesome, can you approve this PR if it all looks good?

**Comment by quartzmo** [comment]
@billyjacobson Can this change be merged into master and released?

**Review by billyjacobson — APPROVED** [review]

**Pull request merged**